arch/arm/src/kinetis/hardware/kinetis_usbhs.h

```diff
@@ -1,54 +1,38 @@
 /********************************************************************************************************************
  * arch/arm/src/kinetis/hardware/kinetis_usbhs.h
  *
- *   Copyright (C) 2018 Gregory Nutt. All rights reserved.
- *   Author: Gregory Nutt <gnutt@nuttx.org>
- *           Ramtin Amin <ramtin@lambdaconcept.com>
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This header cannot be changed.  We cannot remove Ramtin Amin's copyright claim without permission!